views/drones/list.hbs

```diff
@@ -1 +1,19 @@
-<h2>Available drones</h2>
\ No newline at end of file
+<h2>Available drones</h2>
+
+<div class = "container">
+    {{#each myDrones}}
+        <div class= "container2">
+            <p> <h4>Name:</h4>{{this.name}}</p>
```

Comment:
Using headings is better suited to give a title for the start of a document/section.

I think here it would be better suited to use `<b>` or `<span>` with a class so you can style it accordingly.
```
<p> <span class="specs-name">Name:</span>{{this.name}}</p>